libimage/load.go

```diff
@@ -24,15 +24,25 @@ type LoadOptions struct {
 
 // doLoadReference does the heavy lifting for LoadReference() and Load(),
 // without adding debug messages or handling defaults.
-func (r *Runtime) doLoadReference(ctx context.Context, ref types.ImageReference, options *LoadOptions) (images []string, transportName string, err error) {
+func (r *Runtime) doLoadReference(ctx context.Context, ref types.ImageReference, options *LoadOptions) (names []string, transportName string, err error) {
 	transportName = ref.Transport().Name()
+	var images []*Image
 	switch transportName {
 	case dockerArchiveTransport.Transport.Name():
-		images, err = r.loadMultiImageDockerArchive(ctx, ref, &options.CopyOptions)
+		names, err = r.loadMultiImageDockerArchive(ctx, ref, &options.CopyOptions)
 	default:
 		images, err = r.copyFromDefault(ctx, ref, &options.CopyOptions)
+		for _, img := range images {
+			if len(img.Names()) > 0 {
+				names = append(names, img.Names()...)
```

Comment:
All of the names, incl. names of all the deduplicated images which were not present in the archive?